app/controllers/relationships_controller.rb

```diff
@@ -0,0 +1,23 @@
+class RelationshipsController < ApplicationController
+  before_action :logged_in_user
+
+  attr_reader :user
+
+  def create
+    @user = User.find_by id: params[:followed_id]
+    current_user.follow user
+    respond_to do |format|
+      format.html{redirect_to user}
+      format.js
+    end
+  end
+
+  def destroy
+    @user = Relationship.find_by(id: params[:id]).followed
```

Comment:
khoong tìm thấy thằng relation này thì sao? code đoạn này sẽ báo lỗi undefine for nil class đó